lfsapi/endpoint.go

```diff
@@ -11,11 +11,12 @@ const UrlUnknown = "<unknown>"
 
 // An Endpoint describes how to access a Git LFS server.
 type Endpoint struct {
-	Url            string
-	SshUserAndHost string
-	SshPath        string
-	SshPort        string
-	Operation      string
+	Url                string
+	SshUserAndHost     string
+	SshPath            string
+	SshPort            string
+	Operation          string
+	StandaloneTransfer string
```

Comment:
I don't think this extra `StandaloneTransfer` field is needed. It looks like it's used below to return fake Batch API responses, when that case is already handled in the `TransferQueue`:

https://github.com/git-lfs/git-lfs/blob/5703b808220103063ededf5c6149c93201be9e07/tq/transfer_queue.go#L412-L421

I think you can use the UrlConfig object during the initialization of the `tq.Manifest` instead, like the check you added in `getVerifyStateFor()`.

https://github.com/aleb/git-lfs/blob/4949d921400da0b7669dc3467e83a5d25042ac29/tq/manifest.go#L73